blackbox/docs/sql/aggregation.txt

```diff
@@ -371,6 +371,9 @@ corresponding to the percentile of each fraction specified::
     SELECT 1 row in set (... sec)
 
 
+When a query with ``percentile`` function is issued against an empty table
+or it does not match any records in a table then a null result is returned.
```

Comment:
this is not table specific, this is about input rows. so i would explain it that way